packages/storybook/src/react-card.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import readme from '@frameless/components-css/card/README.md?raw';
+import { Card } from '@frameless/components-react/src/Card';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import '@frameless/components-css/card/index.scss';
+
+// adds Card to Storybook
+const meta = {
+  title: 'React Component/Card',
+
+  id: 'react-card',
+
+  component: Card,
+
+  argTypes: {
+    children: {
+      name: 'Content',
+
+      description: 'Card',
+
+      type: {
+        name: 'string',
+        required: true,
+      },
+
+      defaultValue: '',
+    },
+  },
+
+  args: {},
+
+  tags: ['autodocs'],
+
+  parameters: {
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof Card>;
+
+export default meta;
+
+// defines that this (part of the) story is documentation
+type Story = StoryObj<typeof meta>;
+
+// adds Frameless Card section to Card in Storybook
+export const Default: Story = {
+  name: 'Frameless Card', // in sidenav shows Frameless Card below Docs
+  args: {},
+  parameters: {},
```

Comment:
Ik denk dat deze nog `children` nodig heeft om het goed te doen?

Reply:
Huh dit is de oude versie, er zitten wel children in en hij werkt, net als de voorbeeld van Ali. Plus ik heb geen comment meer in die file. Dit is waarschijnlijk een nieuwe versie maar ik heb alles gepushed umm dat is heel raar.